test_scripts/Smoke/Resumption/ATF_Resumption_app_registered_in_more_than_30sec_after_IGN_OFF.lua

```diff
@@ -0,0 +1,110 @@
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app = nil -- will be initialized after application registration
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("SDL does not perform App resumption when app is registered in more than 30 sec after IGNITION_ON.")
+
+function Test:IGNITION_OFF()
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "SUSPEND" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLPersistenceComplete")
+
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "IGNITION_OFF" })
+  EXPECT_NOTIFICATION("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = false }) 
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose"):Do(function ()
+    self.mobileSession:Stop()
+    StopSDL()
+  end)
+end
+
+function Test:Restart_SDL_And_Add_Mobile_Connection()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:Sleep_31_sec()
```

Comment:
@LitvinenkoIra Please remove unused parameter `self` that implicitly pass to function